src/components/ErrorNotification/ErrorNotification.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import classNames from 'classnames';
+import { switchError } from '../../utils/switchError';
+import { Error } from '../../types/Error';
+
+type Props = {
+  error: Error | null;
+};
+
+export const ErrorNotification: React.FC<Props> = ({ error }) => {
+  const errorType = !error
+    ? null
+    : switchError(error);
+
+  return (
+    <div
+      className={classNames(
+        'notification',
+        { 'is-warning': errorType === 'NoPostsYet' },
```

Comment:
The conditions for the className `is-warning` and `is-danger` are mutually exclusive. Consider improving readability by using an else clause or a ternary operator.